dockers/piped-base/Dockerfile

```diff
@@ -27,4 +27,8 @@ RUN \
     /installer/install-terraform.sh && \
     # Delete installer directory.
     rm -rf /installer && \
-    rm -f /var/cache/apk/*
+    rm -f /var/cache/apk/* && \
+    addgroup -S -g 1000 pipecd && \
+    adduser -S -u 1000 -G pipecd -h /home/pipecd pipecd
+
+USER pipecd
```

Comment:
I think we should move lines 31-34 to before installing the tools (before line 11).

Reply:
I see, gonna give it a try.

Reply:
Seems to be unable to perform apk with pipecd user. It may be good to keep the order as is and just add `-p` option to mkdir command.
```
ERROR: Unable to lock database: Permission denied
ERROR: Failed to open apk database: Permission denied
```

Reply:
I could see the owner of the home directory is pipecd even if created in the build phase with root permission.
```
/ $ ls -la /home/pipecd/
total 20
drwxr-sr-x    1 pipecd   pipecd        4096 Oct 23 01:47 .
drwxr-xr-x    1 root     root          4096 Oct 23 01:47 ..
-rw-------    1 pipecd   pipecd          44 Oct 23 01:48 .ash_history
drwxr-xr-x    2 root     root          4096 Oct 23 01:47 tools
```